conan/internal/model/conan_file.py

```diff
@@ -386,8 +387,11 @@ def run(self, command: str, stdout=None, cwd=None, ignore_errors=False, env="",
         wrapped_cmd = command_env_wrapper(self, command, env, envfiles_folder=envfiles_folder)
         from conan.internal.util.runners import conan_run
         if not quiet:
-            ConanOutput().writeln(f"{self.display_name}: RUN: {command}", fg=Color.BRIGHT_BLUE)
+            ConanOutput().info(f"{self.display_name}: RUN: {command}", fg=Color.BRIGHT_BLUE)
         ConanOutput().debug(f"{self.display_name}: Full command: {wrapped_cmd}")
+        if quiet or ConanOutput()._conan_output_level == LEVEL_QUIET:
+            stdout = subprocess.DEVNULL if stdout is None else stdout
+            stderr = subprocess.DEVNULL if stderr is None else stderr
```

Comment:
I am not sure, now we have a command that:
- can fail
- it will output almost nothing
- it will not generate any file to inspect (they are generated in a temp folder, but user know nothing at all)

So no messages, no feedback, no files... I don't know if just the final error message will be enough to troubleshoot when something goes wrong. 
And the command does a full ``conan install`` of dependencies, many things can go wrong...
At least the folder with the generated files should be printed in case of error and probably not sending this to DEVNULL, but instead redirecting to a file inside the temp folder, and the error message of the exception pointing to this log file as well.

Reply:
I've marked this as a look-into, I think it would be good to dedicate some time next week to discuss how the output of this command should behave.